stories/DataTable/expandable/groupBy.stories.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import * as _ from 'lodash-es';
+
+import DataTable from '../../../src/DataTable/DataTable';
+import Row from '../../../src/DataTable/TableRow';
+import tableDataItems from '../../constants/sampleMovieData';
+
+const ExpandedComponent = ({ data, rowProps }) => {
+	let rowPropsPlus = {
+		...rowProps,
+		expandableRows: false,
+		columns: [
+			{
+				id: 0,
+				name: 'ID',
+				selector: row => row.id,
+				sortable: true,
+				wrap: true,
+			},
+			{
+				id: 1,
+				name: 'Title',
+				selector: row => row.title,
+				sortable: true,
+				wrap: true,
+			},
+			{
+				id: 2,
+				name: 'Year',
+				selector: row => row.year,
+				sortable: true,
+				wrap: true,
+			},
+			{
+				id: 3,
+				name: 'Runtime',
+				selector: row => row.runtime,
+				sortable: true,
+				wrap: true,
+			},
+		],
+	};
+	return _.map(data.groupData, item => {
+		return (
+			<div key={item.title}>
```

Comment:
this `div` is not required, we can pass the `key` attribute to `Row` component
PS. make sure that the key value is unique (`item.title` might have duplicate values)
Also, `item.title` might be undefined.

A good idea might be to check out how datatable generate the `key` value and use the same mechanism for generating keys for group rows.